app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -26,36 +31,57 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
   return (
     <Formik
       initialValues={note}
-      validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
+      validationSchema={VALIDATION_SCHEMA}
       onSubmit={handleSubmit}
     >
       {({ isSubmitting }) => (
-        <FormikForm className="w-full">
+        <FormikForm noValidate className="w-full">
           <Pane.Body className="space-y-6">
             <Input
               required
               className="w-full flex-grow-0"
-              label="Title"
+              label={t("common.title")}
               name="title"
+              placeholder={t("placeholder.title")}
             />
             <Textarea
               required
               className="w-full flex-grow-0"
-              label="Description"
+              label={t("common.description")}
               name="description"
-              rows={8}
+              placeholder={t("placeholder.description")}
+              rows={1}
+            />
+            <Select
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label={t("common.assignedContact")}
+              name="assignedContact"
+              options={ASSIGNED_CONTACTS}
+              placeholder={t("placeholder.selectRole")}
```

Comment:
The `name` is using the keyword `contact`. But the placeholder is using the keyword `role`. Let's stick onto  one of them.